snapshot/overlay/overlay_test.go

```diff
@@ -34,7 +33,7 @@ func TestOverlay(t *testing.T) {
 }
 
 func TestOverlayMounts(t *testing.T) {
-	ctx := namespaces.WithNamespace(context.Background(), "snapshotter-overlay-test")
+	ctx := context.TODO()
```

Comment:
I reverted the PR which put namespacing all the way down into the snapshotter implementations. This was just part of that revert, `Background` should probably be used instead of `TODO`, but I didn't touch the result.